src/app-layout/visual-refresh-toolbar/drawer/index.tsx

```diff
@@ -100,17 +101,9 @@ export function AppLayoutDrawerImplementation({ appLayoutInternals }: AppLayoutD
             variant="icon"
           />
         </div>
-        {toolsContent && (
-          <div
-            className={clsx(
-              styles['drawer-content'],
-              activeDrawerId !== TOOLS_DRAWER_ID && styles['drawer-content-hidden']
-            )}
-          >
-            {toolsContent}
```

Comment:
This special treatment for the `toolsContent` has a reason. I think this page is broken with the change: https://github.com/cloudscape-design/components/blob/main/pages/app-layout/runtime-drawers-imperative.page.tsx